incubator/awx/templates/web_deployment.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "awx.fullname" . }}-{{ .Values.webName }}
+  labels:
+    app: {{ template "awx.name" . }}
+    chart: {{ template "awx.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    component: {{ .Values.webComponent }}
+spec:
+  replicas: {{ .Values.webReplicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "awx.name" . }}
+      release: {{ .Release.Name }}
```

Comment:
Add `component` label.

Reply:
@unguiculus

I changed everything as you said. (Add component label.)

I pushed my sources.

Please check the changes.